docs/shadow_config_spec.md

```diff
@@ -165,6 +165,13 @@ virtual hosts, so each worker can only advance according to the propagation
 delay to avoid dependency violations. Therefore, not all threads will have 100%
 CPU utilization.
 
+#### `general.progress`
+
+Default: false  
+Type: Bool
+
+Show the simulation progress.
```

Comment:
"Show the simulation progress on stderr." ?

Maybe also worth saying a little about the differing behavior based on whether stderr is a tty or not.

Reply:
Expanded the comments.